### ISSUE TITLE
ANOTHER server crash when using the Quantum Quarry [1.10.2]

### ISSUE THREAD

**Issue by hugeblank** [rwtema/extrautilities_old_issues#1288]
[crash-2017-02-04_22.46.35-server.txt](https://github.com/rwtema/extrautilities/files/752626/crash-2017-02-04_22.46.35-server.txt)
This crash occurs on loop when you place a hell biome marker in an overworld quarry. This is on a 1.10.2 server with multiple other mods, so I'm not ruling out the fact that this may be a combined issue with another mod. Nevertheless, the log has been pasted above, for you to investigate. Unfortunately this has taken down an entire server, due to it being chunkloaded, and crash looping.

**Comment by YOYOK9** [comment]
I am Having this same issue. (@hugeblank delete dimension -9999 in the world folder to get back on your server) 
My report: http://pastebin.com/fdhgd6CA
Extra stuff:
extrautils2-1.10.2-1.2.0
Mc: 1.10.2
SpongeForge: spongeforge-1.10.2-2202-5.1.0-BETA-2057

**Comment by InjuredWolf** [comment]
Mod Idea: Yet Another Server Crash (YASC) lol

**Comment by tenten8401** [comment]
Same problem.
https://hastebin.com/meqomapavo.scala

**Comment by hugeblank** [comment]
@tenten8401 @YOYOK9 this issue may get fixed with the latest update of spongeforge (2117), posted ~4 hours ago. 
@InjuredWolf billion dollar idea there, my friend!

**Comment by hugeblank** [comment]
@YOYOK9 -9999 is the Deep Dark. -(5 digit number that I cannot recall) is the "workhouse dimension", where the quarries pull from. And I've tried deleting that workhouse dimension... and it crashes still.
@InjuredWolf billion dollar Idea there!
@tenten8401 And you had 4 players online! that's gotta suck. 
Shameless plug... Links to my servers [discord](https://discord.gg/yzrWhNr) and [Technic](http://www.technicpack.net/modpack/edit/929560/main) are here... and there. It's referred to as Project 
Beta. (Note: It's not in beta... and it hasn't been for over 3 months now) IP is dannysmc.com:25585 should you want to join :P

**Comment by Brend-Smits** [comment]
Any update on this issue?

**Comment by hugeblank** [comment]
Obviously not :/

**Comment by woodzykiler** [comment]
can you plz DROP the shitty quantum quarry and give us back the enderquarry plz!

**Comment by ktole1999** [comment]
its not a quarry issue from i have read. its a sponge issue which was just updated and supposedly fixed. quit bashing the mod when its not the mods fault.

**Comment by hugeblank** [comment]
yeah I'm with @ktole1999 here, the quantum quarry is nothing bad, it's just a bug (that has supposedly been resolved) that made it a pain in the backside to manage. 

I really don't think it's anyone's place to bash a mod that is created on open source terms, for free. He can do whatever he damn well wants to do with the mod and there isn't anything we could say that would/should even remotely effect the content of the mod.

**Comment by mzx987** [comment]
Hopefully this gets some attention. Recently discovered this same bug on my SpongeForge server. Luckily after the crash, it was rolled back a few seconds to prevent the server from crashing every time on start up.
